Packs/SecurityScorecard/ReleaseNotes/1_0_7.md

```diff
@@ -0,0 +1,18 @@
+
+#### Integrations
+
+##### SecurityScorecard
+
+- %%UPDATE_RN%%
```

Comment:
Please fill this out with the added commands. See https://xsoar.pan.dev/docs/documentation/release-notes#enhancement-examples as an example.